concepts/observability-configurable-logging/05-custom-fluentbit-operator.md

```diff
@@ -114,64 +114,116 @@ To configure Fluent Bit, the user must create a new CR regarding to the CRD of t
-* Fluent Bit pods and the used image should be managed by Kyma
-* User-provided configuration parts should be as close as possible to the Fluent Bit configuration format
-* Filters can be used to
-  * Parse workload specific log formats (for instance, multi-line exceptions)
+The following requirements for the telemetry operator should be considered when choosing a sufficient Fluent Bit configuration:
+* Fluent Bit pods and the used image should be managed by Kyma.
+* User-provided configuration parts should be as close as possible to the Fluent Bit configuration format.
+* Filters can be used for the following purposes:
+  * Parse workload-specific log formats (for example, multi-line exceptions)
   * Include or exclude parts of the logs (ship only a specific namespace to a backend)
   * Prepare the log metadata to comply with the backend's requirements (de-dotting for Elastic)
 
-### Tags managed by the telemetry-operator
+### Scenario 1: Tags managed by the telemetry operator
 
 Properties:
-* The user-defined LogPipeline elements must not contain any "match" attributes
-* Pipeline elements that emit new tags (for instance rewrite_tag filters) must not be used
-* The telemetry-operator adds ad "match" attribute to all filters and outputs
+* The user-defined LogPipeline elements must not contain any "match" attributes.
+* Pipeline elements that emit new tags (for example, `rewrite_tag` filters) must not be used.
+* The telemetry operator adds a "match" attribute to all filters and outputs.
 
-Consequences:
-* Fill control over resource consumption since no additional buffers can be created
-* Design goal to allow pasting existing Fluent Bit config samples is partially lost
+Advantages:
+* Full control over resource consumption because no additional buffers can be created.
 
-### Managed tag per pipeline
+Disadvantages:
+* Design goal to allow pasting existing Fluent Bit config samples is partially lost.
+
+### Scenario 2: Managed tag per pipeline
 
 Properties:
-* The telemetry-operator provides a specific tag for each LogPipeline that has to be consumed by the pipeline sections
-  * The telemetry-operator creates either a dedicated input plugin or buffered rewrite_tag filter to isolate the pipeline
-* There is a defined way to consume the operator-provided log stream (for instance a tag placeholder or documented naming pattern)
-* The LogPipeline can emit new tags using the rewrite_tag filter
-* Potential overhead since complex pipelines have to be split to multiple simple pipelines, each having an own buffer
+* The telemetry operator provides a specific tag for each LogPipeline that must be consumed by the pipeline sections.
+* There is a defined way to consume the operator-provided log stream (for example, a tag placeholder or documented naming pattern).
+* The LogPipeline can emit new tags using the `rewrite_tag` filter.
+* Potential overhead because complex pipelines must be split into multiple simple pipelines, each having its own buffer.
```

Comment:
Is that a property or a disadvantage?

Reply:
A disadvantage

Reply:
But we ...prefer to keep it under "Properties", not "Disadvantages" ? 🤔

Reply:
Good catch, moved it